src/preprocessing/passes/ackermann.cpp

```diff
@@ -187,9 +187,117 @@ void collectFunctionsAndLemmas(FunctionToArgsMap& fun_to_args,
 
 /* -------------------------------------------------------------------------- */
 
+/* Update the statistics for each uninterpreted sort */
+void updateUSortsCardinality(USortToBVSizeMap& usort_cardinality, TNode term)
+{
+  TypeNode type = term.getType();
+  if (type.isSort())
+  {
+    if (usort_cardinality.find(type) == usort_cardinality.end())
+    {
+      usort_cardinality.insert(make_pair(type, make_pair(0, 0)));
+    }
```

Comment:
The above `if` statement is not necessary: When you do `usort_cardinality[type]` and the key `type` does not exist, C++ will create that element for you, initialized with the default value for the type (which happens to be a pair of two zeros).

Reply:
For readability, I would either include the `if` statement or explain this default `c++` behavior inline.